src/pymovements/datasets/codecomprehension.py

```diff
@@ -98,6 +101,8 @@ class CodeComprehension(DatasetDefinition):
 
     name: str = 'CodeComprehension'
 
+    long_name: str = 'Code Comprehension Dataset'
```

Comment:
I would be in favor of using lower case for dataset here, as it not really a part of the name.

For example in DIDEC or PoTeC the Corpus is part of the name, but in cases where it's not it should be lowercase (e.g. as with GazeBase).

Reply:
done